Tests/DatadogTests/Datadog/Tracing/TracingFeatureTests.swift

```diff
@@ -99,6 +86,13 @@ class TracingFeatureTests: XCTestCase {
 
         let payload = server.waitAndReturnRequests(count: 1)[0].httpBody!
 
+        // Expected payload format:
```

Comment:
Is this asserted somehow ?

Reply:
Yes, the `SpanMatcher's` expects exactly this format in: `try SpanMatcher.fromNewlineSeparatedJSONObjectsData (payload)`. It throws an error if the array of `SpanMatchers` cannot be build.

Reply:
Same will be done later for RUM payload:
```swift
let rumMatchers = try RUMEventMatcher.fromNewlineSeparatedJSONObjectsData(payload)
```
For now I build RUM payload assertions around the raw data, because we don't yet have `RUMEventMatcher` ready.